hazelcast/src/main/java/com/hazelcast/security/UsernamePasswordCredentials.java

```diff
@@ -85,7 +85,7 @@ public int getClassId() {
 
     @Override
     public String toString() {
-        return "UsernamePasswordCredentials [name=" + getName() + "]";
+        return "UsernamePasswordCredentials{name=" + name + "}";
```

Comment:
groupconfig password id represented as `*`, and here password is not printed at all. Can we make them similar, either remove all or print `*`.

Reply:
for security reasons it is better to not print password.

Reply:
Removed the password from GroupConfig's toString